sway-lib-std/src/storage.sw

```diff
@@ -681,6 +681,262 @@ impl<V> StorageVec<V> {
     pub fn clear(self) {
         let _ = clear::<u64>(__get_storage_key());
     }
+
+    /// Swaps two elements.
+    ///
+    /// ### Arguments
+    ///
+    /// * element1_index - The index of the first element.
+    /// * element2_index - The index of the second element.
+    ///
+    /// ### Reverts
+    ///
+    /// * If `element1_index` or `element2_index` is greater than the length of the vector.
+    ///
+    /// ### Storage Access
```

Comment:
```suggestion
    /// ### Number of Storage Accesses
```
And similarly throughout the rest of this file. I also noticed [this](https://github.com/FuelLabs/sway/blob/ee0a0fd9ea0f401904cd3dbff95268a3c7bf21fb/sway-lib-std/src/storage.sw#L243) which would be nice to fix here if you can.